test_util/dcos_api_session.py

```diff
@@ -110,7 +110,7 @@ def __init__(self, dcos_url: str, masters: list, public_masters: list,
         self.slaves = sorted(slaves)
         self.public_slaves = sorted(public_slaves)
         self.all_slaves = sorted(slaves + public_slaves)
-        self.zk_hostports = ','.join(':'.join([host, '2181']) for host in self.public_masters)
+        self.zk_hostports = "zk-1.zk:2181,zk-2.zk:2181,zk-3.zk:2181,zk-4.zk:2181,zk-5.zk:2181"
```

Comment:
PTAL. I've updated the accompanying enterprise PR accordingly here https://github.com/mesosphere/dcos-enterprise/pull/502/files#diff-99053e08b3a194291f27808e1dabbcaaR23